### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [okikio/bundle#20] [General]
# index

A quick and easy way to bundle, minify, and compress (gzip and brotli) your ts, js, jsx and npm projects all online, with the bundle file size.

https://bundlejs.com/

**Comment by okikio** [comment]
Hey, check out the new comment section.

**Reply by okikio** [reply]
Reply

**Comment by okikio** [comment]
*bundlejs* now has a new logo. 

![android-96x96](https://user-images.githubusercontent.com/17222836/168493676-bb83727b-b926-4559-8cca-75048bde55f3.png)
